modules/catalog/src/main/java/org/apache/ignite/internal/catalog/CatalogServiceImpl.java

```diff
@@ -1089,4 +644,195 @@ private static void throwUnsupportedDdl(String msg, Object... params) {
     interface UpdateProducer {
         List<UpdateEntry> get(Catalog catalog);
     }
+
+    private static CatalogSchemaDescriptor getSchema(Catalog catalog, @Nullable String schemaName) {
```

Comment:
all this new lines have nothing to do with the problem described in ticket, so, please, move it to another patch